DEBIAN/control

```diff
@@ -0,0 +1,12 @@
+Package: openrefine
+Version: 2.8
+Maintainer: Mauricio Vargas <mvargas@dcc.uchile.cl>
+Section: utils
+Source: OpenRefine
+Homepage: http://openrefine.org/
+Architecture: amd64
```

Comment:
I think the architecture should not be restricted to `amd64`, as it should run independently anywhere with a JVM, no?